vuepress/docs/v7.3/docs/consume/identity-management.md

```diff
@@ -65,3 +69,151 @@ Keycloak enables One Time Passwords (OTP) login to Entando Applications. See [Ke
+
+## Options
+### Configuring Keycloak with an External Oracle DBSM
+
+This example shows the method to enable the Entando Operator to auto-provision external Oracle databases for all of its needs, including Keycloak, PORTDB, SERVDB, and DEDB. 
+
+The process requires the reformulation of the custom resources for the App Engine, Keycloak server and database service. Replace the variables as required by your application, paying special attention to variables with the prefix `YOUR-`.
+
+1. Update the EntandoApp custom resource, using your namespace and application name. 
+``` yaml
+apiVersion: entando.org/v1
+kind: EntandoApp
+metadata:
+  namespace: YOUR-NAMESPACE
+  name: YOUR-APP-NAME
+spec:
+  dbms: none
+  ingressHostName: YOUR-APP-NAME.k8s-entando.org
+  standardServerImage: tomcat
+  environmentVariables:
+    - name: MAX_RAM_PERCENTAGE
+      value: "75"
+    - name: PORTDB_URL
+      value: jdbc:oracle:thin:@//*****************************:1521/*******
+    - name: PORTDB_USERNAME
+      valueFrom:
+        secretKeyRef:
+         key: username
+         name: YOUR-PORTDB-SECRET
+    - name: PORTDB_PASSWORD
+      valueFrom:
+        secretKeyRef:
+         key: password
+         name: YOUR-PORTDB-SECRET
+    - name: PORTDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleValidConnectionChecker
+    - name: PORTDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleExceptionSorter
+    - name: SERVDB_URL
+      value: jdbc:oracle:thin:@//*********************************:1521/******
+    - name: SERVDB_USERNAME
+      valueFrom:
+        secretKeyRef:
+         key: username
+         name: YOUR-SERVDB-SECRET
+    - name: SERVDB_PASSWORD
+      valueFrom:
+        secretKeyRef:
+         key: password
+         name: YOUR-SERVDB-SECRET
+    - name: SERVDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleValidConnectionChecker
+    - name: SERVDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleExceptionSorter
+    - name: SPRING_DATASOURCE_URL
+      value: jdbc:oracle:thin:@//*************:1521/********
+    - name: SPRING_DATASOURCE_USERNAME
+      valueFrom:
+        secretKeyRef:
+         key: username
+         name: YOUR-DEDB-SECRET
+    - name: SPRING_DATASOURCE_PASSWORD
+      valueFrom:
+        secretKeyRef:
+         key: password
+         name: YOUR-DEDB-SECRET
+    - name: SPRING_JPA_DATABASE_PLATFORM
+      value: org.hibernate.dialect.OracleDialect
+    - name: SERVDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleValidConnectionChecker
+    - name: SERVDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.oracle.OracleExceptionSorter
+  replicas: 1
+  resourceRequirements:
```

Comment:
@jyunmitch I'd remove the `resourceRequirements` property from here, just because it's not working now.

Reply:
will do

Reply:
@pietrangelo what about on the [EntandoApp CR page](https://developer.entando.com/v7.3/docs/reference/entandoapp-cr.html), should I remove the  resourceRequirements from there too?